src/leap/common/events/catalog.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# catalog.py
+# Copyright (C) 2015 LEAP
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful",
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+"""
+Events catalog.
+"""
+
+
+EVENTS = [
+    "CLIENT_SESSION_ID",
+    "CLIENT_UID",
+    "IMAP_CLIENT_LOGIN",
+    "IMAP_SERVICE_FAILED_TO_START",
+    "IMAP_SERVICE_STARTED",
+    "IMAP_UNHANDLED_ERROR",
+    "KEYMANAGER_DONE_UPLOADING_KEYS",
+    "KEYMANAGER_FINISHED_KEY_GENERATION",
+    "KEYMANAGER_KEY_FOUND",
+    "KEYMANAGER_KEY_NOT_FOUND",
+    "KEYMANAGER_LOOKING_FOR_KEY",
+    "KEYMANAGER_STARTED_KEY_GENERATION",
+    "MAIL_FETCHED_INCOMING",
```

Comment:
yes, it is. the fetch functionality has been moved to its own place for mail 0.4, it doesn't belong to imap anymore.